### PR TITLE
Vt 13

### PR DESCRIPTION
**Pull request by LINDVIOR18**
Add password encoder and fix some bugs!

### CODE REVIEW

File: src/main/java/com/vesta/service/impl/UserServiceImpl.java [hunk 1]
```diff
@@ -63,18 +74,31 @@ public void delete(Long id) {
...
-    public Token login(AccountCredential accountCredential) {
-        if (userRepository.existsByUsernameOrEmailAndPassword(
-                accountCredential.getUsername(),
-                accountCredential.getEmail(),
-                accountCredential.getPassword()))
-            return tokenService.generatedToken(accountCredential.getUsername());
+    @Override
+    public Map<String, Token> login(AccountCredential accountCredential) {
+
+        UserEntity userEntity = userRepository
```
Comment (ccaraian) [inline src/main/java/com/vesta/service/impl/UserServiceImpl.java:87]: I think this is common with line 79

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -10,4 +10,7 @@ spring.flyway.url=jdbc:postgresql://localhost:5432/postgres
 spring.flyway.password=postgres
 spring.flyway.user=postgres
 
-swagger.host=localhost:8080
\ No newline at end of file
+swagger.host=localhost:8080
+
+logging.level.root=WARN
```
Comment (ccaraian) [inline src/main/resources/application.properties:15]: change logging level to INFO

File: src/main/java/com/vesta/service/impl/UserServiceImpl.java (hunk 1 above)
Comment (LINDVIOR18) [reply src/main/java/com/vesta/service/impl/UserServiceImpl.java:87]: what do you mean?

Comment (LINDVIOR18) [reply src/main/java/com/vesta/service/impl/UserServiceImpl.java:87]: It was a conflict and i resolved it. I mean

Comment (ccaraian) [reply src/main/java/com/vesta/service/impl/UserServiceImpl.java:87]: This code is the same for the line 79, so which mean you have duplicate code what isn't ok. Extract this code to a private method.
userRepository.findByUsername(accountCredential.getUsername())
                .orElseThrow(() -> new NotFoundException("The username or email doesn't exist"))

**Pull request closed without merging**